internal/testrunner/runners/system/runner.go

```diff
@@ -1118,7 +1120,10 @@ func (r *runner) setupService(ctx context.Context, config *testConfig, serviceOp
 	}
 
 	serviceDeployer, err := servicedeployer.Factory(serviceOptions)
-	if err != nil {
+	if errors.Is(err, os.ErrNotExist) {
+		return nil, svcInfo, nil
```

Comment:
It could be added a logger Debug message here to note that no service is going to be setup.

```suggestion
	if errors.Is(err, os.ErrNotExist) {
		logger.Debug("skip setting service, no deployer defined")	
		return nil, svcInfo, nil
```